python/ray/scripts/scripts.py

```diff
@@ -1437,6 +1437,13 @@ def rsync_up(cluster_config_file, source, target, cluster_name, all_nodes):
     default=False,
     help="If True, the usage stats collection will be disabled.",
 )
+@click.option(
+    "--extra-screen-args",
```

Comment:
If `extra screen args` is passed, it would be good to check 
that `screen` is True, else `cli_logger.abort("To use `extra_screen_args`, it is required to use the `--screen flag`")`

Reply:
Added the check.